### ISSUE TITLE
PostgreSQL installation bug

### ISSUE THREAD

**Issue by lorddaedra** [opencart/opencart#8047]
PHP 7.4 with pgsql extension

```
Fatal error: Uncaught Exception: 
Error: 
Could not load database adaptor pgsql! in /home/user/opencart-test.domain.com/system/library/db.php:33 
Stack trace: 
#0 /home/user/opencart-test.domain.com/install/model/install/install.php(4): DB->__construct('pgsql', 'localhost', 'backwket_openca...', 'password...', 'user_openca...', '5432') 
#1 /home/user/opencart-test.domain.com/system/engine/loader.php(248): ModelInstallInstall->database(Array) #2 /home/user/opencart-test.domain.com/system/engine/proxy.php(47): Loader->{closure}(Array, Array) 
#3 /home/user/opencart-test.domain.com/install/controller/install/step_3.php(11): Proxy->__call('database', Array) 
#4 /home/user/opencart-test.domain.com/system/engine/action.php(79): ControllerInstallStep3->index() 
#5 /home/user/opencart-test.domain.com/system/engine/router.php(67): Action->execute(Object(Registry)) 
#6 /home/user/opencart-test.domain.com/system/engine/router.php(56): Rout in /home/user/opencart-test.domain.com/system/library/db.php on line 33
```

**Comment by danielkerr** [comment]
Fixed in master

**Comment by prabhatkumaroc082** [comment]
fix #9110
